tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -83,10 +83,16 @@ elif [ $BUILD_DEVICE = "gpu" ]; then
         $GET_DOCKER_IMAGE_CMD --repository "onnxruntime-$IMAGE" \
             --docker-build-args="--build-arg BASEIMAGE=nvcr.io/nvidia/cuda:11.1.1-cudnn8-devel-${BUILD_OS} --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} --build-arg INSTALL_DEPS_EXTRA_ARGS=\"${INSTALL_DEPS_EXTRA_ARGS}\" --build-arg USE_CONDA=${USE_CONDA} --network=host" \
             --dockerfile Dockerfile.ubuntu_gpu_training --context .
-elif [ $BUILD_DEVICE = "tensorrt" ]; then
-        # TensorRT container release 20.12
-        IMAGE="$BUILD_OS-cuda11.1-cudnn8.0-tensorrt7.2"
-        DOCKER_FILE=Dockerfile.ubuntu_tensorrt
+elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
+        if [ $BUILD_DEVICE = "tensorrt-v7.1" ]; then
+            # TensorRT container release 20.07
+            IMAGE="$BUILD_OS-cuda11.0-cudnn8.0-tensorrt7.1"
+            DOCKER_FILE=Dockerfile.ubuntu_tensorrt7_1
+        else
+            # TensorRT container release 20.12
+            IMAGE="$BUILD_OS-cuda11.1-cudnn8.0-tensorrt7.2"
+            DOCKER_FILE=Dockerfile.ubuntu_tensorrt
+        fi
```

Comment:
@snnn , FYI line 96 looks odd. Looks like it was introduced by https://github.com/microsoft/onnxruntime/pull/7632
This is an Ubuntu image, so it shouldn't have added build args for manylinux?

Reply:
Oh, you are right. Sorry about that. The things added by me can be removed.

Reply:
@snnn, thanks for confirming. @chilo-ms  , can you remove all those extra build args related to CentOS?

Reply:
#8089

Reply:
Thanks, I think I'll wait #8089 merged to master first, so Changming doesn't need to resolve the conflict.

Reply:
seems like 8089 is still a draft PR.
just merge yours first?

Reply:
Okay, it's going to be only a small conflict for 8089 to resolve.